prog/vm/host_nexus.rb

```diff
@@ -245,8 +245,16 @@ def get_boot_id
   end
 
   def available?
-    sshable.cmd("true")
-    true
+    ssh_session = sshable.start_fresh_session
+    is_available = false
+
+    begin
+      vm_host.perform_health_checks(ssh_session)
+    ensure
+      ssh_session&.close
+    end
+
+    is_available
```

Comment:
This is a big change in behavior, slowing down `available?` quite a bit, by losing connection caching.

Reply:
fixed the issue thanks